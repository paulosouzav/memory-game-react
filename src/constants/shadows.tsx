import { css } from 'styled-components';
import { TRANSPARENT_BLACK } from './colors';

export const SHADOWS = Object.freeze({
  none: css`
    box-shadow: none;
  `,
  soft: css`
    box-shadow: 0 0 8px ${TRANSPARENT_BLACK.soft};
  `,
  normal: css`
    box-shadow: 0 0 8px ${TRANSPARENT_BLACK.normal};
  `,
  hard: css`
    box-shadow: 0 0 8px ${TRANSPARENT_BLACK.hard};
  `,
});
