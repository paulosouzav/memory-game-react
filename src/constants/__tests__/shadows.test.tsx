import { SHADOWS } from '../shadows';

describe('SHADOWS', () => {
  it('should have expected values', () => {
    expect(SHADOWS).toMatchSnapshot();
  });
});
