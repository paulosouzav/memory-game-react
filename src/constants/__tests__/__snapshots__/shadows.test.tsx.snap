// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SHADOWS should have expected values 1`] = `
Object {
  "hard": Array [
    "
    box-shadow: 0 0 8px ",
    "rgba(0,0,0,0.7)",
    ";
  ",
  ],
  "none": Array [
    "
    box-shadow: none;
  ",
  ],
  "normal": Array [
    "
    box-shadow: 0 0 8px ",
    "rgba(0,0,0,0.5)",
    ";
  ",
  ],
  "soft": Array [
    "
    box-shadow: 0 0 8px ",
    "rgba(0,0,0,0.2)",
    ";
  ",
  ],
}
`;
